HelpSource/Classes/String.schelp

```diff
@@ -992,4 +993,3 @@ method::bounds
 Tries to return a link::Classes/Rect:: with the size needed to fit this string if drawn with given font.
 argument:: font
 A link::Classes/Font::
-
```

Comment:
Do we need this change? (it's minor, I know)

Reply:
Definitely not needed, thanks for pointing that out!